src/main/java/com/wcc/platform/utils/FileUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.wcc.platform.utils;
+
+import org.apache.logging.log4j.util.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+public class FileUtil {
+
+    private static final Logger logger = LoggerFactory.getLogger(FileUtil.class);
+
+    public static String readFileAsString(String fileName) {
+        var classLoader = FileUtil.class.getClassLoader();
+
+        try {
+            InputStream inputStream = classLoader.getResourceAsStream(fileName);
+            if (inputStream != null) {
+                BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
```

Comment:
If we have the usecase to read a file with multiple lines ( like a csv or txt which ends with something like \n ) then worth having this utility method otherwise it will become very inefficient to convert a very very large json file to a string.